<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>helpWindow</class>
 <widget class="QDialog" name="helpWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>641</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton{
background-color:   rgb(50, 50, 50);
text-align: left;
font-family: &quot;Georgia&quot;;
font-weight: bold;
font-size: 14px;
color: rgb(232, 232, 232);
border: none ;
border-bottom: 1px solid rgb(255, 85, 0) ;
icon: url(:/Icons/Icons/OpenCloseImage.svg);
}

QPushButton:hover{
color: rgba(232, 232, 232,100);
border-bottom-color: rgba(255, 85, 0,100) ;
icon: url(:/Icons/Icons/OpenCloseImage_Hover.svg);
}


#helpWindow{
background-color: rgb(50, 50, 50);
}

QScrollArea{
border:none;
}

QScrollBar:vertical{
border: none;
background-color: rgb(50,50,50);
width:14px;
margin:14px 0 14px 0;
border-radius:0px;
}

QScrollBar::handle:vertical{
background-color: rgb(198, 66, 0);
min-height: 30px;
border-radius: 7px;
}


QScrollBar::handle:vertical:hover{
background-color: rgba(198, 66, 0,60%);
}


/* Button top */
QScrollBar::sub-line:vertical{
border:none;
background-color: rgb(50,50,50);
height:15px;
border-top-left-radius: 7px;
border-top-right-radius: 7px;
subcontrol-position:top;
subcontrol-origin: margin;
}

QScrollBar::sub-line:vertical:hover{
background-color: rgba(198, 66, 0,40%);
}

QScrollBar::sub-line:vertical:pressed{
background-color: rgb(185,0,92);
}

/* Button bot */

QScrollBar::add-line:vertical{
border:none;
background-color: rgb(50,50,50);
height:15px;
border-bottom-left-radius: 7px;
border-bottom-right-radius: 7px;
subcontrol-position: b	ottom;
subcontrol-origin: margin;
}

QScrollBar::add-line:vertical:hover{
background-color: rgba(198, 66, 0,60%);
}

QScrollBar::add-line:vertical:pressed{
background-color: rgba(185,0,92);
}

/*Reset arrow*/
QScrollBar::up-arrow:vertical, QScrollBar::down-arrow:vertical{
background:none;
}

QScrollBar::add-page:vertical, QScrollBar::sub-page:vertical{
background:none;
}





</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <widget class="QWidget" name="IntroBlockWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="OpneIntroductionButton">
        <property name="text">
         <string>Introduction</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAsNeeded</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>623</width>
           <height>72</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(232, 232, 232);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item alignment="Qt::AlignTop">
           <widget class="QLabel" name="IntroductionLabel">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;The software application is aimed at assisting the student in learning methods for solving systems of nonlinear equations. A calculator will be used for learning in the application, in which the solution of the system is written step by step, testing of the student's knowledge with the help of the presented examples and a theoretical block for each method of decision.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="CalculatorBlockWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="OpenCalculatorButton">
        <property name="text">
         <string>Calculator</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="lineWidth">
         <number>1</number>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::AdjustIgnored</enum>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>609</width>
           <height>1525</height>
          </rect>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(232, 232, 232);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLabel" name="CalculatorLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;body&gt;This section will discuss the operation of the calculator.For the calculator to work correctly and get the result, you need to enter five parameters: the accuracy of the solution (see Figure 1.1), the method of solving a system of nonlinear equations (see Figure 1.2), the number of unknowns in equations (see Figure 1.3), the system of equations (see Figure 1.4). values ​​of unknowns (see Figure 1.5).&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures1_1">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/Accuracy_picture1.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures1_1">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 1.1 - Accuracy of the solution &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures1_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/MethodSolutionSystem.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures1_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 1.2 - Methods of solving a system of nonlinear equations&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures1_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/NumberOfUknows.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures1_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 1.3 - Number of unknowns in equations&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures1_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/SystemOfEquations.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures1_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 1.4 - System of equations&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures1_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/InitialValues.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures1_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 1.5 - Initial values ​​of unknowns&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;To enter the system of equations correctly, you need to use special symbols. For example, in the program, unknowns are indicated only by &amp;quot;x&amp;quot;, ie if you need to enter a second unknown, you need to write &amp;quot;x2&amp;quot;. An example of system introduction is shown in Figure 1.6.&lt;/p&gt;&lt;p&gt;The calculator supports special characters:&lt;/p&gt;&lt;p&gt;- degree (&amp;quot;^&amp;quot;) - 4 ^ 2 (four in degree two);&lt;/p&gt;&lt;p&gt;- trigonometric functions (cos, sin, tg, ctg);&lt;/p&gt;&lt;p&gt;- logarithmic functions (log (a, x), lg (x), ln (x)).&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures1_6">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/ExampleSystemOfEquations.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures1_6">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 1.6 - Example of system introduction&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="TheoryBlockWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="OpenTheoryButton">
        <property name="text">
         <string>Theory</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_3">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_3">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>609</width>
           <height>276</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(232, 232, 232);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="TheoryLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Work with the theory takes place on the &amp;quot;Theory&amp;quot; tab. To view the theory from the method, just click on the button with the appropriate name, then a block of text with the theory will expand.&lt;/p&gt;&lt;p&gt;In the lower right corner, there is a button that stores &lt;span style=&quot; font-style:italic;&quot;&gt;opened&lt;/span&gt; theory, in word-format. The appearance of the button is shown in Figure 2.1.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures2_1">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/Icons/Icons/SaveDocFile.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures2_1">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 2.1 - Button to save the theory in word format&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="LobbyBlockWidget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="OpenLobbyButton">
        <property name="text">
         <string>Lobby</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollArea_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents_4">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>609</width>
           <height>1079</height>
          </rect>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(50, 50, 50);
color: rgb(232, 232, 232);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_8">
          <item>
           <widget class="QLabel" name="LobbyLabel">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;There will be only three buttons on the Lobby tab: &amp;quot;Create Lobby&amp;quot;, &amp;quot;Connect to Lobby&amp;quot;, &amp;quot;Create Pack&amp;quot;.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;The &amp;quot;Create Lobby&amp;quot; button creates a room with administrator rights. In the first window of the dialogue, the authorization of the room is performed. The user enters his login and password for the room, an example of which is shown in Figure 3.1. After creation, the user opens a room window where he can download the pack (see Figure 3.2) or open a chat (see Figure 3.3) and correspond. The administrator has the right to run the test after downloading the pack through the start button, which is shown in Figure 3.4. After the start of the test, ie after pressing the start button, all other users of the room will appear on the screen loading equation.&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures3_1">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/AutorizationExample.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures3_1">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 3.1 - Example enters login and password&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures3_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/Lobby/Icons/InstallPackIcon.svg&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures3_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 3.2 - Button for install pack in lobby&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures3_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/Lobby/Icons/ChatIcon.svg&quot;  width='100' height='100'  /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures3_3">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 3.3 - Button for open chat lobby&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures3_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/Lobby/Icons/StartTest.svg&quot; width='100' height='100' /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures3_4">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 3.4 - Button for start test&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;The &amp;quot;Connect to Lobby&amp;quot; button connects to an already created lobby by entering the host's IP address issued through an application that creates a virtual local area network (such as a Radmin VPN). After connecting, the user has the right to: view the list of users, correspond in the chat, respond to the download of the admin packs.&lt;/p&gt;&lt;p align=&quot;justify&quot;&gt;The &amp;quot;Create pack&amp;quot; button opens a window in which you can create a pack of equations, the window shown in Figure 3.5.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="Pictures3_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;img src=&quot;:/helpPicture/Icons/HelpImage/CreatePackExample.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TextPictures3_5">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Picture 3.5 - Example create pack&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
